src/section/mod.rs

```diff
@@ -61,8 +61,6 @@ impl Section {
         let public_key_set = secret_key_set.public_keys();
         let secret_key_share = secret_key_set.secret_key_share(0);
 
-        // Note: `ElderInfo` is normally signed with the previous key, but as we are the first node
-        // of the network there is no previous key. Sign with the current key instead.
```

Comment:
Because it's not true anymore. The elders info is now always signed with the latest key. This makes things simpler I think.

Reply:
Yeah, I agree it will be simpler.
However, do you mean the new_elders_info is now signed by the new section key ?
if so, the section_keys are no longer forms a chain anymore ?

Reply:
They still form a chain, it's just the elders info is now signed by the last key, not the one-before last as before.